package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@springernature/backend-proxy",
-  "version": "2.0.1",
+  "version": "2.1.0",
```

```suggestion
  "version": "2.0.1",
```

The version change happens as part of the release